definitions/subworkflows/cellranger_mkfastq_and_count.cwl

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "running cellranger mkfastq and count"
+
+inputs:
+    bcl_directory:
+        type: string
+    chemistry:
+        type: string
```

Comment:
Since auto is the default, seems like this could be `string?`

Reply:
Yup!